app/views/past_foreign_secretaries/index.html.erb

```diff
@@ -12,736 +12,769 @@
-      <span class="govuk-visually-hidden"> &ndash; </span>
-      <span class="govuk-body-s govuk-!-margin-top-1 govuk-!-margin-bottom-0 term">1791 to 1801</span>
-    </p>
-  </li>
-  <li class="govuk-grid-column-one-quarter">
-    <p class="govuk-heading-s govuk-!-margin-bottom-4">
-      Francis Godolphin Osborne, Marquess of Carmarthen
-      <span class="govuk-visually-hidden"> &ndash; </span>
-      <span class="govuk-body-s govuk-!-margin-top-1 govuk-!-margin-bottom-0 term">1783 to 1791</span>
-    </p>
-  </li>
-  <li class="govuk-grid-column-one-quarter">
-    <p class="govuk-heading-s govuk-!-margin-bottom-4">
-      George Nugent Temple Grenville, Earl Temple
-      <span class="govuk-visually-hidden"> &ndash; </span>
-      <span class="govuk-body-s govuk-!-margin-top-1 govuk-!-margin-bottom-0 term">1783</span>
-    </p>
-  </li>
-  <li class="govuk-grid-column-one-quarter">
-    <p class="govuk-heading-s govuk-!-margin-bottom-4">
-      <a class="govuk-link" href="/government/history/past-foreign-secretaries/charles-fox">Charles James Fox</a>
-      <span class="govuk-visually-hidden"> &ndash; </span>
-      <span class="govuk-body-s govuk-!-margin-top-1 govuk-!-margin-bottom-0 term">1783</span>
-    </p>
-  </li>
-  <li class="govuk-grid-column-one-quarter clear-person">
-    <p class="govuk-heading-s govuk-!-margin-bottom-4">
-      Thomas Robinson, Lord Grantham
-      <span class="govuk-visually-hidden"> &ndash; </span>
-      <span class="govuk-body-s govuk-!-margin-top-1 govuk-!-margin-bottom-0 term">1782 to 1783</span>
-    </p>
-  </li>
-  <li class="govuk-grid-column-one-quarter">
-    <p class="govuk-heading-s govuk-!-margin-bottom-4">
-      <a class="govuk-link" href="/government/history/past-foreign-secretaries/charles-fox">Charles James Fox</a>
-      <span class="govuk-visually-hidden"> &ndash; </span>
-      <span class="govuk-body-s govuk-!-margin-top-1 govuk-!-margin-bottom-0 term">1782</span>
-    </p>
-  </li>
-</ol>
+<%# 21st century section %>
+<div class="govuk-grid-row historic-people-index__section-row-header">
+  <div class="govuk-grid-column-full">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "21st century",
+      padding: true,
+      border_top: 2,
+      font_size: "l"
+    } %>
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  </div>
+</div>
+<div class="historic-people-index__section-row" role="list">
+  <div class="govuk-grid-row">
+    <div role="listitem" class="govuk-grid-column-one-quarter">
+      <%# Custom heading as links to external page %>
+      <h3 class="govuk-heading-s govuk-!-margin-bottom-0">
+        <a class="govuk-link" href="/government/people/dominic-raab/">Dominic Raab</a>
+      </h3>
+      <p class="govuk-body">2019 to present</p>
+    </div>
+    <div role="listitem" class="govuk-grid-column-one-quarter">
```

Comment:
This (and the other PRs) all seem very repetitive. I think there would be value in making hashes of the data then looping through them and using a single partial to populate with the data along the lines of this:

```
<h3 class="govuk-heading-s govuk-!-margin-bottom-0">
      <% if link.present? %>  
         <a class="govuk-link" href="*variable here*">*variable here*</a>
     <% else %>
          *variable here*
    <% end %>
      </h3>
<div role="listitem" class="govuk-grid-column-one-quarter">
      <%= render "govuk_publishing_components/components/heading", {
        text: *variable here*,
        heading_level: 3,
        font_size: "s",
      } %>
      <p class="govuk-body">*variable here*</p>
    </div>
```
You could potentially expand it out to the image card examples as well? I don't think it's common enough to be worth making a component, but it's not at all DRY to be seeing repeated HTML markup like this.

Reply:
As this is hardcode - do you have a good example to ref to use for this context, is this worth adding to `content_item` or locale .[yml](https://github.com/alphagov/whitehall/blob/175fec283eab3efa5e61698d4eb90cf47cefe869/config/locales/en.yml) file?

Reply:
I don't think it fits in a locale file as the strings are not really translatable (names/dates) and they're definitely not page furniture and content items are complicated. I'd keep it in situ as arrays of hashes like: 
```
C21 = [
{name:"Boris Johnson", dates: "2020-2021"}, 
{name:"Theresa May", dates: "xxxx-xxxx", link: "/government/whatever"}, 
]

C21.each do
	call partial
end```
Excuse any bad syntax, but you get the drift?

Reply:
So! This has been split out, added to the controller and the partial iterates over.